src/services/web-monitoring-api.ts

```diff
@@ -0,0 +1,26 @@
+import WebMonitoringDb, {Page} from './web-monitoring-db';
+
+export default class WebMonitoringApi {
+    private configuration = (window as any).webMonitoringConfig;
+    private dbApi = new WebMonitoringDb({
```

Comment:
Initially, I was passing an instance of the dbAPI created in ui.tsx file to a constructor method in webApi, but decided to move it into webAPI object itself. Both approaches seem smelly to me. Not sure which is better, but webApi needs dbApi to work.